local-dev.yml

```diff
@@ -33,7 +33,6 @@ services:
       LINUX_ADMIN_USER_DELETE_USER_IN_USER_ZONE_CMD: '/home/hsuserproxy/delete_user.sh'
       HS_IRODS_PROXY_USER_IN_USER_ZONE: 'localHydroProxy'
       HS_IRODS_USER_ZONE_DEF_RES: 'hydroshareLocalResc'
-      IRODS_ZONE: 'hydroshareZone'
```

Comment:
@sblack-usu I think you mentioned why but I'm not remembering. Why remove `IRODS_ZONE` here?